File: C/Loops/while.c
#include <stdio.h>
#include <stdbool.h>

int main () {
  bool maybe = true;
  int i = 0;

  while (maybe) {
    printf("%d\n", i);

    if (i == 9) {
      maybe = false;
    }

    i++;
  }

  printf("Done.");

  return 0;
}

File: Java/Loops/While.java
public class While {
  public static void main(String[] args) {
    Boolean maybe = true;
    int i = 0;

    while (maybe) {
      System.out.println(i);

      if (i == 9) {
        maybe = false;
        System.out.println("Done.");
      }

      i++;
    }
  }
}
